### PR TITLE
feat(es/lint): implement symbol-description rule

### PR DESCRIPTION
**Pull request by ArturAralin**
**Description:**

feat(es/lint): implement symbol-description rule

**BREAKING CHANGE:**

No

### CODE REVIEW

File: crates/swc_ecma_lints/src/rules/symbol_description.rs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        });
+    }
+}
+
+impl Visit for SymbolDescription {
+    fn visit_call_expr(&mut self, call_expr: &CallExpr) {
+        if let Callee::Expr(expr) = &call_expr.callee {
+            if let Expr::Ident(ident) = expr.as_ref() {
+                if !self.is_symbol_call(ident) {
+                    return;
```
Comment (kdy1) [inline crates/swc_ecma_lints/src/rules/symbol_description.rs:113]: I think this will miss

```js
foo(() => Symbol())
```

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thank you!

File: crates/swc_ecma_lints/src/rules/symbol_description.rs (hunk 1 above)
Comment (ArturAralin) [reply crates/swc_ecma_lints/src/rules/symbol_description.rs:113]: Yes you right. Thx

**Review by kdy1 — APPROVED** [review]
Thank you!

---

swc-bump:
 - swc_ecma_lints

**Pull request merged**